app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java

```diff
@@ -213,7 +234,6 @@ private void createAddressView() {
      * Syncing registered participant to Firebase Firestore (field array "participantId")
      */
     public void registerToActivity(View v) {
-        userId = fAuth.getCurrentUser().getUid();
```

Comment:
Thanks for removing those redundant definitions of `userId`!

Reply:
Thanks for pointing them out!